nextcord/automod.py

```diff
@@ -0,0 +1,365 @@
+
+
+class AutoModerationRule(Hashable):
+    """Represents an auto moderation rule.
+
+    .. container:: operations
+        .. describe:: x == y
+            Check if two auto moderation rules are equal.
+        .. describe:: x != y
+            Check if two auto moderation rules are not equal.
+
+    Attributes
+    -----------
+    id: :class:`int`
+        The auto moderation rule's ID.
+    guild: :class:`Guild`
+        The guild that this auto moderation rule is active in.
+    name: :class:`str`
+        The name of this auto moderation rule.
+    event_type: :class:`EventType`
+        The type of the event that the rule should execute when this event is called.
+    trigger_type: :class:`TriggerType`
+        The trigger type of this auto moderation rule.
+    guild_id: :class:`int`
+        The guild ID that this auto moderation rule is active in.
+    creator_id: :class:`int`
+        The creator of this auto moderation rule's ID.
+    enabled: :class:`bool`
+        Whether this rule is enabled or not.
+    exempt_role_ids: List[:class:`int`]
+        A list of roles that will not be affected by this rule.
+        .. note::
+            Bots are always not affected by any rule.
+    exempt_channel_ids: List[:class:`int`]
+        A list of channels that will not be affected by this rule.
+    keyword_filters: Optional[List[:class:`str`]]
+        The custom filters for this auto moderation rule. `None` if not set.
+    presets: Optional[List[:class:`KeywordPresetType`]]
+        The pre-set types of this auto moderation rule. `None` if not set.
+    actions: List[:class:`AutoModerationAction`]
+        The actions that this auto moderation rule will execute if triggered.
+    """
+
+    def __init__(self, *, state: ConnectionState, guild: Guild, data: AutoModerationRulePayload):
+        self.id: int = int(data["id"])
+        self.guild: Guild = guild
+        self._state: ConnectionState = state
+        self.guild_id: int = int(data["guild_id"])
+        self.name: str = data["name"]
+        self.creator_id: int = int(data["creator_id"])
+        self.event_type: EventType = try_enum(EventType, data["event_type"])
+        self.trigger_type: TriggerType = try_enum(TriggerType, data["trigger_type"])
+        self.enabled: bool = data["enabled"]
+        self.exempt_role_ids: List[int] = [int(exempt_role) for exempt_role in data["exempt_roles"]]
+        self.exempt_channel_ids: List[int] = [
+            int(exempt_channel) for exempt_channel in data["exempt_channels"]
+        ]
+        trigger_metadata: TriggerMetadataPayload = data["trigger_metadata"]
+        self.keyword_filters: Optional[List[str]] = trigger_metadata.get("keyword_filter")
+        self.actions: List[AutoModerationAction] = []
+        self.presets: Optional[List[KeywordPresetType]] = [try_enum(KeywordPresetType, preset) for preset in trigger_metadata["presets"]] if trigger_metadata.get("presets") is not None else None  # type: ignore
```

Comment:
Yea, let's not. Please make this statement multiline

Reply:
Also surely you can use `if "string" in dict` over checking if .get is None